auth0/resource_auth0_connection_test.go

```diff
@@ -37,3 +36,50 @@ resource "auth0_connection" "my_connection" {
 	}
 }
 `
+
+func TestAccConnectionWaad(t *testing.T) {
+
+	resource.Test(t, resource.TestCase{
+		Providers: map[string]terraform.ResourceProvider{
+			"auth0": Provider(),
+		},
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccConnectionWaadConfig,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("auth0_connection.my_connection", "name", "Acceptance-Test-Connection"),
+					resource.TestCheckResourceAttr("auth0_connection.my_connection", "strategy", "waad"),
+				),
+			},
+		},
+	})
+}
+
+const testAccConnectionWaadConfig = `
+provider "auth0" {
+	domain = ""
+	client_id = ""
+	client_secret = ""
```

Comment:
This might be whats breaking the tests as I have the domain, client id and secret defined as environment variables inside wercker. If you do it as [here](https://github.com/yieldr/terraform-provider-auth0/pull/18/files#diff-f1f4f442734b7f84b82b24ef27028b60L30) it should pick the right one up.